MekHQ/src/mekhq/gui/adapter/UnitTableMouseAdapter.java

```diff
@@ -244,60 +311,43 @@ public void actionPerformed(ActionEvent action) {
                 customsDir.mkdir();
             }
             for (Unit unit : units) {
-                String fileName = unit.getEntity().getChassis() + " "
-                        + unit.getEntity().getModel();
-                try {
-                    if (unit.getEntity() instanceof Mech) {
-                        // if this file already exists then don't overwrite
-                        // it or we will end up with a bunch of copies
-                        String fileOutName = sCustomsDir + File.separator
-                                + fileName + ".mtf";
-                        String fileNameCampaign = sCustomsDirCampaign
-                                + File.separator + fileName + ".mtf";
-                        if ((new File(fileOutName)).exists()
-                                || (new File(fileNameCampaign)).exists()) {
-                            JOptionPane
-                                    .showMessageDialog(
-                                            null,
-                                            "A file already exists for this unit, cannot tag as custom. (Unit name and model)",
-                                            "File Already Exists",
-                                            JOptionPane.ERROR_MESSAGE);
-                            return;
-                        }
-                        FileOutputStream out = new FileOutputStream(
-                                fileNameCampaign);
-                        PrintStream p = new PrintStream(out);
+                String fileName = unit.getEntity().getChassis() + " " + unit.getEntity().getModel();
+                if (unit.getEntity() instanceof Mech) {
```

Comment:
Could this be moved into a method? This massive if...elseif block is unruly as it stands.